testsuite/test_shell.py

```diff
@@ -7,6 +7,26 @@
 from testsuite.support import ROOT_DIR, PseudoFile
 
 
+def safe_line_split(line):
+    """ parses the path, from message details """
+
+    # On certain platforms, the drive notation conflicts
+    # with the message seperator, inducing problems during tuple unpacking.
+
+    split = line.split(':')
+    if 'win' in sys.platform:
+        if len(split) == 5:
+            drive, path, x, y, msg = split
```

Comment:
Agreed, that will be helpful as we continue to build up the windows support and verify that use case.